pkg/nodebootstrap/assets.go

```diff
@@ -1,16 +1,16 @@
-// Code generated by go-bindata. DO NOT EDIT.
```

Comment:
this reverts it to the old bindata version, no longer required after #1420

Reply:
Thanks! I think I still had the old go-bindata locally. Fixed